tests/acceptance/features/coreApiSharePublicLink1/accessToPublicLinkShare.feature

```diff
@@ -1,4 +1,4 @@
-@api @public_link_share-feature-required @files_sharing-app-required @issue-ocis-reva-282
+@api @public_link_share-feature-required @files_sharing-app-required @issue-ocis-reva-282 @issue-4623 @notToImplementOnOCIS
```

Comment:
IMHO you can remove the whole feature file instead of adding `@notToImplementOnOcis` tag. As we are removing all scenarios from ocis those are tagged with `@notToImplementOnOcis` and `@skipOnOcis`

Reply:
this feature will be implemented in ocis in the future. So i skipped this for now.

@phil-davis can you suggest what will be the best option? Should we delete not implemented occ command related scenario or skip for now?

Reply:
we already have this tag `@toImplementOnOCIS`. But do this scenarios fail due to the occ command?

Reply:
yes bro if we don't have occ command for techPreview enabled then how can we run tests for such tests case

Reply:
Maybe the preview is enabled by default in the oCIS, like the default `Shares` folder.